packages/mds-compliance-engine/engine/count_processors.ts

```diff
@@ -60,13 +54,13 @@ export function isCountRuleMatch(
  * never matches for a subsequent rule, then it is considered in violation of the policy.
  */
 export function processCountPolicy(
-  policy: Policy,
+  policy: CountPolicy,
   events: (VehicleEvent & { telemetry: Telemetry })[],
   geographies: Geography[],
   devices: { [d: string]: Device }
 ): ComplianceEngineResult | undefined {
   if (getPolicyType(policy) !== RULE_TYPES.count) {
-    throw new UnsupportedTypeError(`${getPolicyType(policy)} submitted to count processor`)
+    throw new UnsupportedTypeError(`${getPolicyType(policy) as string} submitted to count processor`)
   }
```

Comment:
This block should be logically unnecessary now, assuming that what's fed in here is definitely a `CountPolicy` (which is what the types now indicate)